<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QQ Chat Analyzer</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>961</width>
     <height>681</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>文件(&amp;File)</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_read">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>50</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>开始导入</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_browse">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>50</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_file">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>54</y>
       <width>531</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_file">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>导入文件</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>102</y>
       <width>871</width>
       <height>31</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_content">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>871</width>
       <height>471</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;欢迎使用QQ聊天记录分析器！&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;主要功能：&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;1.分析群聊中成员活跃度；&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;2.分析群聊中指定成员活跃时间；&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;3.比较群聊指定成员活跃时间；&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;4.分析私聊双方聊天数量以及时间；&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;5.按时间或关键字搜索聊天内容。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;请点击上方的导入按钮导入【&lt;span style=&quot; color:#ff0000;&quot;&gt;文本格式&lt;/span&gt;】(txt)的聊天记录。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;导入的过程可能&lt;span style=&quot; color:#ff0000;&quot;&gt;非常缓慢&lt;/span&gt;（根据文件总长度），请耐心等待。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;若之前已经导入过聊天记录，可直接使用上面的【查找】、【分析】等功能，&lt;span style=&quot; color:#ff0000;&quot;&gt;无需重复导入&lt;/span&gt;。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>查找(&amp;Search)</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>输入关键字</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_search">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>34</y>
       <width>471</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_search_content">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>30</y>
       <width>121</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>按内容查找</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_search_user">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>30</y>
       <width>121</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>按用户查找</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_search">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>871</width>
       <height>511</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>分析(&amp;Analyze)</string>
    </attribute>
    <widget class="QTableView" name="tableView_analyze">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>40</y>
       <width>451</width>
       <height>581</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>411</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>活跃度分析</string>
     </property>
     <widget class="QPushButton" name="pushButton_ana_1">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>开始分析</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>351</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>分析样本内每个成员的总消息数量。</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>411</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>活跃时间分析</string>
     </property>
     <widget class="QPushButton" name="pushButton_ana_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>开始分析</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>351</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>分析样本内指定成员的活跃时间。</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ana_1">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>成员</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>350</y>
       <width>411</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>活跃时间对比</string>
     </property>
     <widget class="QPushButton" name="pushButton_ana_3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>130</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>开始分析</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>351</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>对比样本内指定两个成员的活跃时间。</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ana_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>成员1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ana_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>成员2</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>高级(Ad&amp;vanced)</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>421</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>QQ聊天记录分析器</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
